### DISCUSSION TITLE
I am a Home-schooled student, who is located in Pakistan. I want the GitHub Student Dev Pack because I am enrolled into 2 curriculums and 1 degree of Computer Science

### DISCUSSION THREAD

**Discussion by onceauser66** [community/community#62502] [GitHub Education]
### Hi everyone,

Hi everyone,
### **I am a home-schooled  really need the GitHub Student Dev Pack; I'm enrolled in 3 curricula!** 

My Curricula:

1. The freeCodeCamp Curriculum
2. The Odin Project
3. Khan Academy: Computer Science

I have proof that I am really learning these courses. So I request the pack if possible. Thank you in advance!

**Comment by Awrsha** [comment]
Unfortunately GitHub's Student Developer Pack is only available to currently enrolled college/university students. Based on their eligibility requirements:

You must be currently enrolled as a student at an accredited high school, college or university.

As proof of enrollment, you'll need to provide your student ID, registration paperwork, or tuition invoice.

Since you're homeschooled, you likely don't have a student ID or official enrollment documents from a school.

While I understand your desire to gain access to the student benefits, GitHub has set clear eligibility requirements to prevent abuse of the program. They need a way to verify that applicants are bona fide students.

A few suggestions for alternative options:

See if any of your curriculum programs offer their own student benefits that you could take advantage of. freeCodeCamp, The Odin Project, etc. may have partnerships with GitHub or other companies.

Consider enrolling part-time at a community college just to gain official student status. Even taking one or two low-cost online courses could provide you with a student ID and make you eligible for GitHub Student.

Explain your situation directly to GitHub and ask if an exception could be made for homeschooled students. They may decline, but it's worth a polite request.

Do your best with free and open source tools for now. You can still learn and build an impressive portfolio without GitHub Student pack. Focus on the knowledge, not the perks.

Ultimately, GitHub has decided to restrict their program to traditionally enrolled students for practical reasons. While unfortunate, I'd encourage you to find creative ways to advance your learning within the bounds of their current policies.

**Reply by onceauser66** [reply]
What about the Government School of Pakistan, or Alison University. Thanks in advance!
